test/functional/outputs/azuremonitor/forward_to_azuremonitor_test.go

```diff
@@ -60,7 +60,6 @@ var _ = Describe("Forwarding to Azure Monitor Log ", func() {
 		nanoTime, _ := time.Parse(time.RFC3339Nano, timestamp)
 		message := "This is my new test message"
 		var appLogTemplate = functional.NewApplicationLogTemplate()
-		appLogTemplate.LogSource = logging.InfrastructureSourceContainer
```

Comment:
Are we not pushing logsource in the datamodel?

Reply:
`LogSource` was removed in [PR 2525](https://github.com/openshift/cluster-logging-operator/pull/2525/files#diff-87df0e8c62be9fc2a545c2cf34f3511c1fd14e7520f7fbd59bfdbd08f466534d):